xapian-applications/omega/query.cc

```diff
@@ -2022,9 +2040,15 @@ eval(const string &fmt, const vector<string> &param)
 		}
 		break;
 	    }
-	    case CMD_pack:
-		value = int_to_binary_string(string_to_int(args[0]));
+	    case CMD_pack: {
+		int temp;
+		if (!parse_signed(args[0].c_str(), temp)) {
+		    throw "NUMBER parameter for pack command"
+			  " must be an integer";
```

Comment:
Pretty much universally in existing Xapian code, when splitting string like this the space goes in the first half not the second.  I don't think it's a documented rule, but I think it makes sense to follow that - when there's a pattern it's easier to spot an exception to it - e.g. when the space gets omitted, or included twice.

Reply:
fixed in a fixup commit.